tajo-catalog/tajo-catalog-common/src/test/java/org/apache/tajo/catalog/TestSchema.java

```diff
@@ -147,13 +147,26 @@ public final void testGetColumnString() {
 		assertEquals(col3, schema.getColumn("addr"));
 	}
 
-	@Test
-	public final void testAddField() {
```

Comment:
This test looks to verify Schema.addColumn() with an unqualified column name. Why don't you add a new test instead of modifying an existing test?